app/controllers/finance/participants/profiles_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Finance
+  module Participants
+    class ProfilesController < BaseController
+      def edit
+        @profile = user.participant_profiles.find(params[:id])
+      end
+
+      def update
+        profile = user.participant_profiles.find(params[:id])
+        Participants::Update.new(profile).call(profile_attributes)
+
+        return redirect_to finance_participant_path(user) if profile.errors.none?
+
+        @profile = profile
```

Comment:
you can delete this line if you set in on 11 then use it throughout the rest of this method